README.md

```diff
@@ -7,6 +7,12 @@ These Intel Data Center hardware features currently include:
 - **Intel® Data Center GPU Flex Series**
 - **Intel® QuickAssist Technology (Intel® QAT)**
 
+| Feature | Supported OCP Version | 
```

Comment:
How about putting the table in the supported platform section?

Reply:
We can definitely move it to supported platform section but I am thinking it might be better here in the root readme right next to supported features since the logical next question from customer is what ocp verison is supported for each feature. This would be important information and should be at the top. I am open either way. :) @chaitanya1731, thoughts?